app/views/friends/_form.html.erb

```diff
@@ -36,9 +36,9 @@
     <%= form.text_field :twitter, class: "form-control" %>
   </div>
   <div class="field form-group">
-    <%= form.text_field :user_id, id: :friend_user_id, class: "form-control", value: current_user.id,type: :hidden %>
+    <%= form.text_field :user_id, id: :friend_user_id, class: "form-control", value: current_user.id, type: :hidden %>
```

Comment:
Why adding hidden field here? Does not it seem to be a potential security break?